### ISSUE TITLE
Please update ruapehu_combi plotting script on vulkan

### ISSUE THREAD

**Issue by rumachan** [rumachan/FITS_plotting#1]
@yannikbehr 

Modifed files are ruapcombi.py and ruapcombi.cfg

**Comment by rumachan** [comment]
Sorry don't remember at all. Lets look at it next week

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by yannikbehr** [comment]
Yes, of course, not urgent at all. I just reverted to the previous working version.
